crates/proof-of-sql/src/sql/proof/provable_query_result_test.rs

```diff
@@ -451,22 +449,3 @@ fn we_can_convert_a_provable_result_to_a_final_result_with_mixed_data_types() {
     .unwrap();
     assert_eq!(res, expected_res);
 }
-
-#[test]
-fn we_cannot_convert_a_provable_result_with_invalid_string_data() {
-    let values = ["abc".as_bytes(), &[0xed, 0xa0, 0x80][..], "de".as_bytes()];
-    let cols: [Box<dyn ProvableResultColumn>; 1] = [Box::new(values)];
-    let column_fields = vec![ColumnField::new("a1".parse().unwrap(), ColumnType::VarChar)];
-    let indexes = Indexes::Sparse(vec![0]);
-    assert!(ProvableQueryResult::new(&indexes, &cols)
-        .to_owned_table::<Curve25519Scalar>(&column_fields)
-        .is_ok());
-    let indexes = Indexes::Sparse(vec![2]);
-    assert!(ProvableQueryResult::new(&indexes, &cols)
-        .to_owned_table::<Curve25519Scalar>(&column_fields)
-        .is_ok());
-    let indexes = Indexes::Sparse(vec![1]);
-    assert!(ProvableQueryResult::new(&indexes, &cols)
-        .to_owned_table::<Curve25519Scalar>(&column_fields)
-        .is_err());
-}
```

Comment:
Removed since we no longer allow arbitrary `ProvableResultColumn` in construction of `ProvableQueryResult` outside tests